File: Level 022/homework/hw.py
# 1) შექმინთ 2 სია  , პირველი სია ინქება ცარიელი  ხოლო მეროე სია ინქება სახელებით სავსე მინიმუმ 5 , თქვენი დავალებაა ამ სიიდან  ჩაამოტომ მეორე სიაში სახელელბი რომელიც  4 სიმბოლოზე ნაკლებია

my_list = ["nika", "dachi", "giorgi", "guga", "nikusha"]
my_list1 = []
for i in my_list:
    if len(i) < 4:
        my_list1.append(i)
   
print(my_list1)
print(my_list) 

File: Level 095/homework/hw.js
const fun = (my_list) => {
    let result_list = []
    for (let i = 0; i < my_list.length; i++) {
        if (my_list[i] % 2 !== 0) {
            result_list.push(my_list[i])
        }
    }
    return result_list
}

console.log(fun([12, 233, 4594, 112, 111]))